packages/api-client/src/conversation/ConversationAPI/ConversationAPI.ts

```diff
@@ -959,12 +959,11 @@ export class ConversationAPI {
   }
 
   /**
-   * Add users to an existing conversation.
+   * Add qualified members to an existing conversation.
```

Comment:
i dont believe qualified members is the correct description as this is just a userId array and not a qualified id array. In general the naming in this file is hard to differentiate for me.